tests/test_tomo.py

```diff
@@ -49,14 +49,85 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+import lzma
 import numpy as np
-from tike.tomo import *
+import os
+import pickle
+import tike.tomo
+import unittest
 
 __author__ = "Daniel Ching"
 __copyright__ = "Copyright (c) 2018, UChicago Argonne, LLC."
 __docformat__ = 'restructuredtext en'
 
 
+class TestPtychoRecon(unittest.TestCase):
+    """Test various ptychography reconstruction methods for consistency."""
+
+    def create_dataset(self, dataset_file):
```

Comment:
The projection data **is** loaded from the disk. This function is not a test. It is "only called with setUp detects that `dataset_file` has been deleted." This function still exists because if we want to regenerate the projection data it is easy to do.